docs/api/materials.md

```diff
@@ -0,0 +1,168 @@
+        "cryopreservation": "Liquid nitrogen",
+        "tissue_histology": "Brain Tumor",
+        "culture_conditions": "Reproducing at a swift rate",
+        "population_doubling_time": "20 Hours",
+        "number_availible_cell_lines": "20"
+    },
+    "contact_user": {
+        "id": "30000000-231f-4dc8-bbfa-02bccfb0372c",
+        "username": "SecondaryProf",
+        "first_name": "Secundus",
+        "last_name": "Profarius",
+        "created_at": "2020-05-08T18:17:39+0000",
+        "updated_at": "2020-05-08T18:17:39+0000"
+    },
+    "created_at": "2020-05-08T18:17:40+0000",
+    "updated_at": "2020-05-08T18:17:40+0000"
+}
+```
+
+## Update an organization
+
+**Request**:
+
+`PUT/PATCH` `/organizations/id`
+
+Parameters:
+
+Name       | Type         |  Description
+-----------|--------------|----------|------------
+category                 | string    | The category of material.
+url                      | string    | The URL that the material was imported from.
+pubmed_id                | string    | The PubMed ID of the associated publication.
+additional_metadata      | json      | Additional details about the material in JSON format.
+title                    | string    | The title of the material.
+needs_mta                | boolean   | Boolean specifying whether the material needs a Material Transfer Agreement to be transferred.
+needs_irb                | boolean   | Boolean specifying whether the material needs an IRB document to be transferred.
+needs_abstract           | boolean   | Boolean specifying whether the material needs a reasearch abstract to be transferred.
+imported                 | boolean   | Boolean specifying whether the material was imported.
+import_source            | string    | String specifying the import source.
+organism                 | string[]  | The organism that the material is associated with. Nullable.
+contact_name             | string    | The associated user. Nullable.
+contact_email            | string    | The associated user's email. Nullable.
+publication_title        | string    | The title of the associated publication. Nullable.
+pre_print_doi            | string    | The DOI of the associated pre-print document. Nullable.
+pre_print_title          | string    | The title of the associated pre-print document. Nullable.
+citation                 | string    | The citation to be used for the material or its publication. Nullable.
+additional_info          | string    | Any additional description of the material or it's publication. Nullable.
+embargo_date             | date      | The date that embargo will be lifted for this material. Nullable.
+contact_user_id          | string    | The ID of the associated user.
+mta_attachment_id        | integer   | The ID of the MTA attachment. Nullable.
+organization_id          | integer   | The ID of the associated organization.
+shipping_requirements_id | integer   | The ID of the shipping requerements. Nullable.
```

Comment:
I'm not entirely sure why, but this and the table above it do not render correctly. It might be because the first column in the data-rows has its first `|` a character before the two lines above them?

Reply:
That was it exactly. Nice catch.